Universal_Robots_Client_Library/CMakeModules/DefineCXX17CompilerFlag.cmake

```diff
@@ -1,27 +1,29 @@
 # DefineCXX17CompilerFlag
 # -----------------------
 #
-# Tries to find the compiler flag which can be used to enable C++17 on the current compiler.
-# If the flag is not found, the macro will issue an cmake error.
+# Tries to find the compiler flag which can be used to enable C++17 on
+# the current compiler.  If the flag is not found, the macro will
+# issue an cmake error.
 #
 # DEFINE_CXX_17_COMPILER_FLAG(<var>)
 #
 # ::
 #
 #   <var>  - variable to store the resulting flag
 #
-# Use this to find the compiler option to enable C++17 compilation. This is usefull
-# on cmake versions / use cases which do not support CXX_STANDARD.
-# Use this in conjunction with target_compiler_option():
+# Use this to find the compiler option to enable C++17
+# compilation. This is usefull on cmake versions / use cases which do
+# not support CXX_STANDARD.  Use this in conjunction with
+# target_compiler_option():
```

Comment:
I'm just curious, did clang format trim the text like this?

Reply:
It's strange that it's trimming around 70-character length. Usually it's 80 or 120, I think. Should be 120 to match MoveIt2, I believe

Reply:
Possibly should just revert the changes to this file

Reply:
> I'm just curious, did clang format trim the text like this?

The `ament_cmake_lint` test complained (all errors are pasted into the commit log):

    Universal_Robots_Client_Library/CMakeModules/DefineCXX17CompilerFlag.cmake:4: Lines should be <= 80 characters long [linelength]

Emacs `M-q` did the actual wrapping; you're right, the `fill-column` variable default is `70`.  I added a `.dir-locals.el` file to set that variable to `80`, matching `ament_cmake_lint`.